spec/data-model/README.md

```diff
@@ -22,39 +22,43 @@ Two equivalent definitions of the data model are also provided:
 
 ## Messages
 
-A `SelectMessage` corresponds to a syntax message that includes _selectors_.
-A message without _selectors_ and with a single _pattern_ is represented by a `PatternMessage`.
+A `SimpleMessage` corresponds to a syntax message that includes _selectors_.
+A message without _selectors_ and with a single _pattern_ is represented by a `ComplexMessage`.
 
 In the syntax,
-a `PatternMessage` may be represented either as a _simple message_ or as a _complex message_,
+a `ComplexMessage` may be represented either as a _simple message_ or as a _complex message_,
 depending on whether it has declarations and if its `pattern` is allowed in a _simple message_.
 
 ```ts
-type Message = PatternMessage | SelectMessage;
+type Message = SimpleMessage | ComplexMessage;
```

Comment:
Why this change in representation? This seems wholly unnecessary, as it links the data model with syntactic details like the representation of a message with a single pattern depending on whether it starts with a `.` or not.

I would have expected a change like this to have at least some argumentation about why it's being made, and some consideration of how it's impacting data model users and operations.